hugegraph-api/src/main/java/com/baidu/hugegraph/version/ApiVersion.java

```diff
@@ -118,10 +118,11 @@ public final class ApiVersion {
      * [0.66] Issue-1567: Support get schema RESTful API
      * [0.67] Issue-1065: Support dynamically add/remove graph
      * [0.68] Issue-1763: Support adamic-adar & resource-allocation API
+     * [0.69] Issue-1748: Support Cypher query RESTful API
      */
 
     // The second parameter of Version.of() is for IDE running without JAR
-    public static final Version VERSION = Version.of(ApiVersion.class, "0.68");
+    public static final Version VERSION = Version.of(ApiVersion.class, "0.69");
```

Comment:
need to update implement-version: https://github.com/apache/incubator-hugegraph/commit/69ac3cc07279178e0f5c1059123d3e1709298f20#diff-3744a042d40e54198e87c7271da06363141165f2523541c71e81c820486ca87fR156

Reply:
updated